### PR TITLE
Argocd Python Client and FastAPI

### PR DESCRIPTION
**Pull request by jharmon96**
Still down for Go, but I wanted to sketch out how this would look and it was faster for me to just code it in a language I knew.

### CODE REVIEW

File: tatras/tatras/main.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        RETURNS:
+          item_name
+          argo_response
+    '''
+    if item_name not in SERVICE_CATALOG['APPS']:
+        return {'status': "that service ain't allowed fam"}
+
+    f = open('argocd_templates/create-app.json',)
+    app = json.load(f)
+    app['spec']['destination']['namespace'] = "test-namespace"
```
Comment (jharmon96) [inline tatras/tatras/main.py:38]: These values are hardcoded right now, but should ultimately come from the request app's `app_config/app.yaml` file

File: argocd-rest-api/create-app.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (CharlesDLandau) [inline argocd-rest-api/create-app.json]: Ironically simpler in k8s, but I get the idea here

**Review by CharlesDLandau — COMMENTED** [review]
LGTM, I'll look at the go version soon

**Comment by CharlesDLandau** [comment]
@jharmon96  I propose to close this as we captured the same direction in #1

**Pull request closed without merging**